### PR TITLE
Adding a list of IDs and a list of the actual monitor objects

### PR DESCRIPTION
**Pull request by maeghan-porter**
## what
* Adding outputs for a list of the monitor IDs as a well as a list of the actual monitor objects.

## why
* There are cases where you might want to have the IDs instead of the names or the objects in their entirety so you can reference their name and id. I have one such case where I want to know both the name and ID for creating an SLO from the monitor.

### CODE REVIEW

**Review by jamengual — DISMISSED** [review]

File: outputs.tf [hunk 1]
```diff
@@ -3,6 +3,16 @@ output "datadog_monitor_names" {
   description = "Names of the created Datadog monitors"
 }
 
+output "datadog_monitor_ids" {
+  value       = values(datadog_monitor.default)[*].name
```
Comment (aknysh) [inline outputs.tf:7]: these are the same as `datadog_monitor_names`.
Should it be `values(datadog_monitor.default)[*].id`?

File: outputs.tf [hunk 2]
```diff
@@ -3,6 +3,16 @@ output "datadog_monitor_names" {
...
+  description = "IDs of the created Datadog monitors"
+}
+
+output "datadog_monitors" {
+  value       = datadog_monitor.default
+  description = "A list of the actual monitor objects."
```
Comment (aknysh) [inline outputs.tf:13]:
```suggestion
  description = "Datadog monitor outputs"
```

**Review by aknysh — CHANGES_REQUESTED** [review]
@maeghan-porter  thanks for the PR, looks good, please see the comments.

Also, after you fix it, please run the following commands from the repo root

```
make init
make github/init
make readme
```

and commit the changes.

Thank you

File: outputs.tf (hunk 1 above)
Comment (maeghan-porter) [reply outputs.tf]: ah yes, copy pasta fail :) Fixed.

**Review by jamengual — APPROVED** [review]

**Review by aknysh — APPROVED** [review]

**Comment by aknysh** [comment]
thanks @maeghan-porter

**Pull request merged**